### PR TITLE
Improve the explanation surrounding "virtual temperature" and a hypothetical "density temperature"

### PR DESCRIPTION
**Pull request by glwagner**
I thought the description could be improved by being a little more explicit. If I've made a mistake, let me know and I can fix it. I also embellished the "side point" by putting it in a box to emphasize that this is additional information that isn't core to understanding the main thread.

### CODE REVIEW

File: docs/src/Formulation.md [hunk 1]
```diff
@@ -356,24 +356,31 @@ where ``\Pi`` is known as the Exner function
...
+
 ```math
 \begin{equation}
 θ_v = \frac{R_m}{R_d} θ.
 \label{e:virtual_pottemp}
 \end{equation}
 ```
-Some texts distinguish virtual and density (potential) temperatures, where density (potential) temperatures take the mass of condensate into account, whereas virtual (potential) temperatures do not. We always take the mass of any condensate into account in the thermodynamics of moist air and do not make this distinction here.
+
+!!! note "Relationship between virtual temperature and 'density temperature'"
```
Comment (glwagner) [inline docs/src/Formulation.md:379]: @navidcy check it out

**Review by trontrytel — APPROVED** [review]
LGTM!

The need to express everything in temperatures...

**Comment by glwagner** [comment]
Any idea why tests fail? Something about a variable `test_project_toml_formatting` that's not defined. It seems unrelated to this PR.

**Comment by charleskawczynski** [comment]
> Any idea why tests fail? Something about a variable `test_project_toml_formatting` that's not defined. It seems unrelated to this PR.

Yeah, it's unrelated, Aqua's `test_project_toml_formatting` [was removed](https://github.com/JuliaTesting/Aqua.jl/blob/34a1a02c1bd13f2273c8dbd9b533acb20144c57e/CHANGELOG.md?plain=1#L61). I'll open a PR to fix this. Also, this PR LGTM.

**Comment by charleskawczynski** [comment]
Feel free to rebase and merge

**Comment by glwagner** [comment]
> The need to express everything in temperatures...

😅

Worth noting, I was reading this section because (some) bulk formula compute heat flux using the air-sea difference in virtual temperature. Blame the fluxes...

**Comment by glwagner** [comment]
> Feel free to rebase and merge

Maybe I'm missing something, but as far as I can tell this branch is up to date with origin/main. What do I need to do?

**Comment by glwagner** [comment]
Note I also changed `Aqua.type_piracy` to `Aqua.type_piracies` via https://github.com/JuliaTesting/Aqua.jl/pull/230

**Comment by charleskawczynski** [comment]
Can we rebase / squash this PR?

**Comment by glwagner** [comment]
Superceded by #174

**Pull request closed without merging**